packages/devtools_app/lib/src/extensions/embedded/view.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '_view_desktop.dart' if (dart.library.html) '_view_web.dart';
+import 'controller.dart';
+
+class EmbeddedExtensionView extends StatelessWidget {
```

Comment:
Doc comment?
Can you explain or reference explanation of difference between embedded and non-embedded extension?